addon/components/document-view.js

```diff
@@ -126,7 +126,6 @@ export default class DocumentViewComponent extends Component {
           count: files.length,
         }),
       );
-      this.afterUpload();
```

Comment:
I think this should be a call to `refreshDocumentList`
Just a missed rename from https://github.com/projectcaluma/ember-alexandria/commit/223febe67c6475e6d91f8d746e6ca6e465b4c1ed

Reply:
Ah, that makes sense. Fixed!